From 633c27bba7e6bda2e0b8c82a3673dd8de38ca393 Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <olteanv@gmail.com>
Date: Mon, 29 Jun 2020 01:58:59 +0300
Subject: [PATCH 14/23] ts2phc_slave: print master offset

Make this information more visible by default, since it is the key
output of this program.

Upstream-Status: Pending

Signed-off-by: Vladimir Oltean <olteanv@gmail.com>
Reviewed-by: Jacob Keller <jacob.e.keller@intel.com>
Signed-off-by: Xiaolei Wang <xiaolei.wang@windriver.com>
---
 ts2phc_slave.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ts2phc_slave.c b/ts2phc_slave.c
index bbda617..435e7d9 100644
--- a/ts2phc_slave.c
+++ b/ts2phc_slave.c
@@ -264,8 +264,8 @@ static int ts2phc_slave_event(struct ts2phc_slave *slave,
	adj = servo_sample(slave->clock->servo, offset, extts_ts,
			   SAMPLE_WEIGHT, &slave->clock->servo_state);

-	pr_debug("%s master offset %10" PRId64 " s%d freq %+7.0f",
-		 slave->name, offset, slave->clock->servo_state, adj);
+	pr_info("%s master offset %10" PRId64 " s%d freq %+7.0f",
+		slave->name, offset, slave->clock->servo_state, adj);

	switch (slave->clock->servo_state) {
	case SERVO_UNLOCKED:
--
2.25.1
